assets/i18n/cy.json

```diff
@@ -215,7 +215,7 @@
   "SELECT_YES_IF_YOU_WANT_TO_CONTINUE_WO_SIGN_IN": "Dewiswch ‘Iawn’ os ydych am fynd yn eich blaen heb fewngofnodi",
   "I_WANT_TO_CONTINUE_WITHOUT_SIGN_IN": "I want to continue without signing in",
   "I_WANT_TO_SIGN_IN": "I want to sign in",
-  "YOU_CAN_CREATE_A_PERSONAL_GOVT": "Os nad oes gennych gyfrif personol ar gyfer Porth y Llywodraeth, gallwch greu un",
+  "YOU_CAN_CREATE_A_PERSONAL_GOVT": "You can create sign in details if you do not have them",
```

Comment:
@Sasankachyuta13 make sure this is updated in the translation xls and translation requested for this. 
Applicable for the below texts as well